### PR TITLE
DEVELOPER-646 Official product downloads lack https URLs and checksums

### PR DESCRIPTION
**Pull request by Dantheman720**
HOLD - Testing checksums.

### CODE REVIEW

**Comment by paulrobinson** [comment]
Let me know when this is ready for review.

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

File: javascripts/terms-and-conditions.js [hunk 1]
```diff
@@ -53,6 +54,13 @@ app.termsAndConditions = {
         $("#tcDownloadFileName").html($.encoder.encodeForHTML(tcDownloadFileName));
       }
 
+    if (tcSrcLink && tcSrcLink.contains(tcDownloadLink) && tcSrcLink.contains("sha256")) {
+        tcSrcLink = $.encoder.canonicalize(window.location.href.substr(window.location.href.indexOf("tcSrcLink=") + 10));
+    }
```
Comment (paulrobinson) [inline javascripts/terms-and-conditions.js]: Why do you need this block of code? Can't 'tcSrcLink' be used as is?

**Comment by Dantheman720** [comment]
This is now updated without the unnecessary JS.  Please let me know if I need to make any other changes.

**Pull request merged**